apps/prairielearn/src/pages/partials/navbarInstructor.ejs

```diff
@@ -70,18 +91,29 @@
             </li>
         <% } %>
     <% } else { %>
-        <% if ((authz_data.has_course_permission_preview || authz_data.has_course_instance_permission_view) && authz_data.course_instances && authz_data.course_instances.length > 0) { %>
-            <li class="navbar-text mx-2 no-select">/</li>
+        <li class="navbar-text mx-2 no-select">/</li>
```

Comment:
There's a behavior change here: this dropdown is now rendered unconditionally, whereas before it was only rendered based on a conditional. Picking apart that conditional:

- `(authz_data.has_course_permission_preview || authz_data.has_course_instance_permission_view)`
  - For `has_course_permission_preview`, the fact that we're even able to *render* a page that would show this dropdown in the first place will mean that the current user has access to this course (I think).
  - For `has_course_instance_permission_view`, given that this is rendered on the _course_ admin page, we should never care about the value of `authz_data.has_course_instance_permission_view`, and in fact I'm pretty sure that will always be null.
- For the `authz_data.course_instances` check, this no longer exists. This does mean that for a course without any course instances, we'll end up rendering a dropdown that won't show any options at all. Maybe we need an explicit empty state on the dropdown?

Reply:
> * For the `authz_data.course_instances` check, this no longer exists. This does mean that for a course without any course instances, we'll end up rendering a dropdown that won't show any options at all. Maybe we need an explicit empty state on the dropdown?

I wrote the other comment before seeing this one. Yes, I think we need an empty state.